api/src/main/java/org/commonjava/maven/galley/model/Transfer.java

```diff
@@ -249,7 +249,7 @@ public void copyFrom( final Transfer f )
         {
             if ( decorator != null )
             {
-                decorator.decorateCopyFrom( f, this );
+                decorator.decorateCopyFrom( f, this, new EventMetadata(  ) );
```

Comment:
another place where a variant of `copyFrom()` may need event metadata eventually